global/iam/introductions-bot.tf

```diff
@@ -12,3 +12,8 @@ resource "aws_iam_role" "delivery-introductions-bot" {
   path = "/services/"
   assume_role_policy = file("${path.module}/templates/assume_role_policy/codebuild.json")
 }
+
+resource "aws_iam_role_policy_attachment" "delivery-introductions-bot" {
+  role =  aws_iam_role.delivery-introductions-bot.name
+  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
```

Comment:
yes, the idea was to get things moving and we can narrow policies once things are functional.

note that the permissions you list would probably go on a separate service role for the introductions-bot which would/could be used as the Lambda execution role.  feel free to issue a PR for that.  I left it out of the initial pass for introductions-bot because I figured you'd probably start with managing that with the Serverless framework.  If you're not using Serverless, then it might make more sense to manage that role with TF.

Reply:
Sure, makes sense. I was just thinking out loud about instituting with permissions boundary at the role level, then letting Serverless or SAM create IAM with least privilege with a maximum permission as defined by the boundary.